src/sonic-config-engine/minigraph.py

```diff
@@ -35,6 +35,8 @@
 VLAN_SUB_INTERFACE_SEPARATOR = '.'
 VLAN_SUB_INTERFACE_VLAN_ID = '10'
 
+backend_asic_interface_name_prefix = 'Ethernet-BP'
```

Comment:
Prefer ALL_CAPS naming for constants/macros.

Reply:
@jleveque Done